packages/webpack/lib/configs/node.js

```diff
@@ -123,7 +123,7 @@ module.exports = function getConfig(config, name) {
       new EnvironmentPlugin({ NODE_ENV: 'development' }),
     ],
     performance: { hints: false },
-    devtool: isProduction ? 'source-map' : 'cheap-module-eval-source-map',
+    devtool: 'inline-source-map',
```

Comment:
Why can't we use an external source-map?

Reply:
Because we are doing in-memory compilation in static mode and the source map will not be found otherwise.